crates/sc-subspace-block-relay/src/consensus.rs

```diff
@@ -91,20 +91,23 @@ struct PartialBlock<Block: BlockT> {
 
 /// The message to the server
 #[derive(Encode, Decode)]
-enum ServerMessage<Block: BlockT, ProtocolRequest> {
+enum ServerRequest<Block: BlockT, ProtocolRequest> {
     /// Initial message, to be handled both by the client
     /// and the protocol
-    InitialRequest(InitialRequest<Block, ProtocolRequest>),
+    Initial(InitialRequest<Block, ProtocolRequest>),
 
     /// Message to be handled by the protocol
-    ProtocolRequest(ProtocolRequest),
+    Protocol(ProtocolRequest),
```

Comment:
Everything here seems to be handled by the protocol :thinking: Is there a better name for it?

Reply:
This looks short/self explanatory, but open to suggestions

Reply:
Well, that is my point. It is not realy explanatory, those requests are protocol requests just like other requests which are not called protocol requests, but clearly still are.

Reply:
Changed